docs/en/development/developer-instruction.md

```diff
@@ -83,6 +83,11 @@ git remote add upstream git@github.com:ClickHouse/ClickHouse.git
 
 After successfully running this command you will be able to pull updates from the main ClickHouse repo by running `git pull upstream master`.
 
+:::tip
```

Comment:
`nit:` Maybe one more tip worth adding is for checking the list of available remotes using `git remote -v` before selecting the remote to use while pushing. It might be useful if we have remote branches tracking the origin and your own fork for example.